.github/workflows/run_e2e_tests.yml

```diff
@@ -99,7 +99,7 @@ jobs:
 
   tear_down_test_executor_instances:
     name: Tear down test executor instances
-    if: always()
+    #if: always() #Uncomment after 3.1.9 version release
```

Comment:
Why are we uncommenting this in 3.1.9

Reply:
We were dealing with flaky tests timing out and giving spurious failures, which needs to be fixed post release.